### DISCUSSION TITLE
Turn on Ultimate mode without screen

### DISCUSSION THREAD

**Discussion by Noir01** [seerge/g-helper#2522] [Q&A]
Hey everyone,

I have a G14. Its screen won’t turn on. I’ve tried connecting to an external display, hard restarts but nothing works. Because of this I think there might be an issue with the graphic card. Before this issue started, I was on eco, so I want to see if turning on ultimate mode would work. Is there any way I could turn it on using just my keyboard?

**Comment by seerge** [comment]
@Noir01 hello. Ultimate mode requires reboot, therefore it’s not possible to turn it w/o interacting with UI.

Usually devices have 2 usb-c ports. One of them is routed to iGPU and another one to dGPU (i.e. same as you would get if you would use Ultimate)

So just try plugging external monitor to each of the ports. If both don’t work - Ultimate won’t help neither
